Analysis/PWGDQ/include/PWGDQCore/VarManager.h

```diff
@@ -129,6 +130,7 @@ class VarManager : public TObject
     kMass,
     kCharge,
     kNBasicTrackVariables,
+    kCosThetaHE,
```

Comment:
Please add this in the group of pair variables, and it has to be before the "kNPairVariables"